content/en/docs/reference/node/seccomp.md

```diff
@@ -0,0 +1,132 @@
+- for a single container using [`spec.containers[*].securityContext.seccompProfile`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context-1)
+- for an (restartable / sidecar) init container using [`spec.initContainers[*].securityContext.seccompProfile`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context-1)
+- for an [ephermal container](/docs/concepts/workloads/pods/ephemeral-containers) using [`spec.ephemeralContainers[*].securityContext.seccompProfile`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context-2)
+
+{{% code_sample file="pods/security/seccomp/fields.yaml" %}}
+
+The Pod in the example above runs as `Unconfined`, while the
+`ephemeral-container` and `init-container` specifically defines
+`RuntimeDefault`. If the ephemeral or init container would not have set the
+`securityContext.seccompProfile` field explicitly, then the value would be
+inherited by the Pod. The same applies to the container, which runs a
+`Localhost` profile `my-profile.json`.
+
+Generally speaking, fields from (ephemeral) containers have a higher priority
+than the Pod level value, while containers which do not set the seccomp field
+are being inherited by the Pod.
+
+{{< note >}}
+It is not possible to apply a seccomp profile to a Pod or container running with
+`privileged: true` set in the container's `securityContext`. Privileged
+containers always run as `Unconfined`.
+{{< /note >}}
+
+The following values are possible for the `seccompProfile.type`:
+
+`Unconfined`
+: The workload runs without any seccomp restrictions.
+
+`RuntimeDefault`
+: A default seccomp profile defined by the
+{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
+is applied. The default profiles aim to provide a strong set of security
+defaults while preserving the functionality of the workload. It is possible that
+the default profiles differ between container runtimes and their release
+versions, for example when comparing those from
+{{< glossary_tooltip text="CRI-O" term_id="cri-o" >}} and
+{{< glossary_tooltip text="containerd" term_id="containerd" >}}.
+
+`Localhost`
+: The `localhostProfile` will be applied, which has to be available on the node
+disk (on Linux it's `/var/lib/kubelet/seccomp`). The availability of the seccomp
+profile is verified by the
+{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
+on container creation. If the profile does not exist, then the container
+creation will fail with a `CreateContainerError`.
+
+### `Localhost` profiles
+
+Seccomp profiles are JSON files following the scheme defined by the
+[OCI runtime specification](https://github.com/opencontainers/runtime-spec/blob/f329913/config-linux.md#seccomp).
+A profile basically defines actions based on matched syscalls, but also allows
+to pass specific values as arguments to syscalls. For example:
+
+{{% code_sample file="pods/security/seccomp/profile.json" %}}
```

Comment:
I'd suggest we embed this sample profile here rather than making it a sample.
A sample is mainly used to k8s resource manifests.

Reply:
Yep, done :+1: